system-design/app-interoperability/application-package-definition.md

```diff
@@ -17,98 +28,186 @@ The application package has the following folder structure:
+**Application Metadata Attributes**
 
-| Attribute       | Type            | Required?       | Description     |
-|-----------------|-----------------|-----------------|-----------------|
-| repoType        | string          | Y               | The type of repository where the deployment definition can be found. Options are: http, git, local. |
-| url             | string          | Y               | The URL to the deployment definition. In case of 'docker-compose', the URL points at the `compose.yaml` file.|
+| Attribute        | Type            | Required?       | Description     |
+|------------------|-----------------|-----------------|-----------------|
+| descriptionFile  | string          | N               | Link to the file containing the application's full description. The file should be a markdown file.|
+| icon             | string          | N               | Link to the icon file (e.g., in PNG format).|
+| licenseFile      | string          | N               | Link to the file that details the application's license. The file should either be a plain text, markdown or PDF file.|
+| releaseNotes     | string          | N               | Statement about the changes for this application's release. The file should either be a markdown or PDF file.|
+| site             | string          | N               | Link to the application's website.|
+| tagline          | string          | N               | The application's slogan.|
+
+**Author Metadata Attributes**
+
+| Attribute        | Type            | Required?       | Description     |
+|------------------|-----------------|-----------------|-----------------|
+| name             | string          | N               | The name of the application's creator.|
+| email            | string          | N               | Email address of the application's creator.|
+
+**Organization Metadata Attributes**
+
+| Attribute        | Type            | Required?       | Description     |
+|------------------|-----------------|-----------------|-----------------|
+| name             | string          | Y               | Organization responsible for the application's development and distribution.|
+| site             | string          | N               | Link to the organization's website.|
+
+**Component Attributes**
+
+| Attribute        | Type            | Required?       | Description     |
+|------------------|-----------------|-----------------|-----------------|
+| cluster          | []ComponentType    | N               | Component type element for indicating the component for deploying the application on a cluster using Helm. See the [Component Type](#component-type-attributes) section below. |
+| standalone      | []ComponentType    | N               | Component type element for indicating the component for deploying the application on a standalone device using Docker compose. See the [Component Type](#compnent-type-attributes) section below. |
+
+**Component Type Attributes**
```

Comment:
"Component Type" is referenced as ComponentType is previous table. Human friendly name might be confusing here.

Reply:
We decided to apply the recommended changes to change this structure so this has been updated.